packages/bpk-component-banner-alert/src/BpkBannerAlertInner.tsx

```diff
@@ -80,9 +80,7 @@ const getIconForType = (
   const Icon = CustomIcon || componentMap[type];
   const AlignedIcon = withButtonAlignment(Icon);
 
-  // TODO: className to be removed
-  // eslint-disable-next-line @skyscanner/rules/forbid-component-props
-  return <AlignedIcon className={className} />;
+  return <span className={className}><AlignedIcon/></span>;
```

Comment:
I wonder as the styling being set is only `fill`'s. whether the classMap could just be a type to fill color?

Reply:
Though it seems `withButtonAlignment` doesn't accept additional props. I wonder why..

Reply:
Hi @metalix2 - were these questions connected to your conversation with Ollie, i.e. do they still apply?